README.md

```diff
@@ -177,18 +229,25 @@ sub-directories are also watched for changes.
 Each new batch of change notifications triggers a fresh evaluation.
 Newly discovered paths are added to the watch list.
 
-## Garbage Collection Roots
+### Garbage Collection Roots
+
+<!-- mdsh TODO: check -->
 
 lorri creates an indirect garbage collection root for each .drv in
 `$XDG_CACHE_HOME/lorri` (`~/.cache/lorri/` by default) each time it
 evaluates your project.
 
 
-## License & Copyright
+### License & Copyright
 
 Copyright 2019 Target
-License: Apache 2.0 (see [`LICENSE` file](./LICENSE))
 
+`$ head -n3 < ./LICENSE`
+```
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+```
```

Comment:
We should stick with the existing text here, as it was requested specifically by Target.